eisbuk-admin/src/components/atoms/OrganizationSettings/OrganizationSettings.tsx

```diff
@@ -0,0 +1,397 @@
+                />
+
+                <Divider />
+                <h5 className={classes.sectionTitle}>Admins</h5>
+                <AdminsField
+                  enableEdit={enableEdit}
+                  currentUser={currentUser}
+                />
+                <Divider />
+                <h5 className={classes.sectionTitle}>Email</h5>
+                <div className={classes.fieldSection}>
+                  {emailFields.map((field) => (
+                    <Field
+                      key={field.name}
+                      label={field.label}
+                      name={field.name}
+                      className={
+                        field.name?.includes("Template")
+                          ? classes.templateField
+                          : classes.field
+                      }
+                      as={TextField}
+                      aria-label={field.label}
+                      variant={enableEdit ? "outlined" : "filled"}
+                      disabled={!enableEdit}
+                      multiline={
+                        field.name?.includes("Template") ? true : false
+                      }
+                      rows={field.name?.includes("Template") ? "4" : "1"}
+                      value={values[`${field.name}`] || ""}
+                    />
+                  ))}
+                </div>
+                <h5 className={classes.sectionTitle}>SMS</h5>
+                <div className={classes.fieldSection}>
+                  <div className={classes.smsFromField}>
+                    <Field
+                      key="smsFrom"
+                      label="SMS From"
+                      name="smsFrom"
+                      className={classes.field}
+                      as={TextField}
+                      aria-label="SMS From"
+                      variant={enableEdit ? "outlined" : "filled"}
+                      disabled={!enableEdit}
+                      value={values.smsFrom || ""}
+                      error={errors.smsFrom}
+                    />
+                    {<ErrorMessage>{errors.smsFrom}</ErrorMessage>}
+                  </div>
+                  {smsFields.map((field) => (
+                    <Field
+                      key={field.name}
+                      label={field.label}
+                      name={field.name}
+                      className={
+                        field.name?.includes("Template")
+                          ? classes.templateField
+                          : classes.field
+                      }
+                      as={TextField}
+                      aria-label={field.label}
+                      variant={enableEdit ? "outlined" : "filled"}
+                      disabled={!enableEdit}
+                      multiline={
+                        field.name?.includes("Template") ? true : false
+                      }
+                      rows={field.name?.includes("Template") ? "4" : "1"}
```

Comment:
Instead of `name.includes("Template")` check (which is repeated) you can simply add `multiline: true` in the FieldProps object (of respective multiline field). And then check that flag for the latter condition: `rows={field.multiline ? "4" : "1"}`

Reply:
Example
```typescript
{
      name: "emailTemplate",
      label: "Email Template",
      multiline: true
}
```